format/index.md

```diff
@@ -173,6 +157,28 @@ Here's how an article (i.e. a resource of type "articles") might appear in a doc
 // ...
 ```
 
+#### Attributes <a href="#document-structure-resource-object-attributes" id="document-structure-resource-object-attributes"></a>
+
+A resource object **MAY** contain additional top-level members. These members
+represent "[attributes]" and may contain any valid JSON value.
+
+Although has-one foreign keys (e.g. `author_id`) are often stored internally
```

Comment:
Earlier, I changed this to match the dasherized recommendation - so `author-id`. Perhaps you're trying to represent how these fields might be stored internally in a database table?